gps/source_manager.go

```diff
@@ -216,7 +218,7 @@ func (sm *SourceMgr) HandleSignals(sigch chan os.Signal) {
 
 				opc := sm.suprvsr.count()
 				if opc > 0 {
-					fmt.Printf("Signal received: waiting for %v ops to complete...\n", opc)
+					sm.logf("Signal received: waiting for %v ops to complete...\n", opc)
```

Comment:
It would clean up a lot to remove it, but how would it affect the user experience?
On the other hand, is there a chance that leaving it to log as before could result in arbitrary test failures?